src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php

```diff
@@ -113,11 +126,10 @@ public function getValidationRuleForCreate(): ParamRuleCollection
 
     /**
      * @param ClaimRequest $claimRequest
-     *
+     * @param int|null $empNumber
      * @return ClaimRequest
-     * @throws InvalidParamException
      */
-    private function setClaimRequest(ClaimRequest $claimRequest): ClaimRequest
+    protected function setClaimRequest(ClaimRequest $claimRequest, ?int $empNumber = null): ClaimRequest
```

Comment:
```suggestion
    protected function setClaimRequest(ClaimRequest $claimRequest, int $empNumber): ClaimRequest
```
better to make this parameter mandatory in next PR